python/lsst/meas/deblender/baseline.py

```diff
@@ -632,24 +751,54 @@ def computeImage(self, cx, cy):
 
 def _fitPsfs(fp, peaks, fpres, log, psf, psffwhm, img, varimg,
              psfChisqCut1, psfChisqCut2, psfChisqCut2b,
-             **kwargs):
-    """!
-    Fit a PSF + smooth background models to a small region around each
-    peak (plus other peaks that are nearby) in the given list of
-    peaks.
-
-    @param[in]      fp           Footprint containing the Peaks to model
-    @param[in]      peaks        a list of all the Peak objects within this Footprint
-    @param[in,out]  fpres        a PerFootprint result object where results will be placed
-    @param[in,out]  log          pex Log object
-    @param[in]      psf          afw PSF object
-    @param[in]      psffwhm      FWHM of the PSF, in pixels
-    @param[in]      img          the Image in which this Footprint finds itself
-    @param[in]      varimg       Variance plane
-    @param[in]      psfChisqCut* floats: cuts in chisq-per-pixel at which to accept
-                                 the PSF model
-
-    Results go into the fpres.peaks objects.
+             **kwargs
+             ):
+    """
+    Fit a PSF + smooth background model (linear) to a small region around each peak.
+    This routine uses a linear least squares algorithm to fit each peak (and neighboring peaks)
+    to the PSF.
+    To make the comparison more robust, each peak is fit with and without including
+    additional terms to recenter the PSF.
+    If a peak matches the recentered PSF, a third fit is calculated by shifting the PSF to the new center.
+    See ``psfChisqCut*`` below for more.
+
+    Parameters
+    ----------
+    fp: `afw.detection.Footprint`
+        Footprint containing the Peaks to model.
+    peaks: `afw.detection.PeakCatalog`
+        Catalog of peaks contained in the parent footprint.
+    fpres: `meas.deblender.PerFootprint`
+        Footprint results object that will hold the results.
+    log: `log.Log`
+        LSST logger for logging purposes.
+    psf: `afw.detection.Psf`
+        PSF of the image.
+    psffwhm: `float`
+        PSF FWHM in pixels.
+    img: `afw.image.ImageF`
+        The image that contains the footprint.
+    varimg: `afw.image.ImageF`
+        The variance of the image that contains the footprint.
+    psfChisqCut*: `float`
```

Comment:
It's a shame that this is basically duplicated text from the above. Given that this is a semi-private (`_`-prefixed) function, I'm not really sure how to fix that though: ideally, I'd suggest you just refer to this description in the `deblend` docstring, but referring to docs on a something that we're discouraging the end-user from looking at seems like poor form.

I guess this is fine as is, unless some better idea strikes you.